code/modules/hydroponics/grown/nettle.dm

```diff
@@ -99,13 +99,13 @@
 	if(..())
 		if(prob(50))
 			user.Paralyze(100)
-			to_chat(user, "<span class='userdanger'>You are stunned by the Deathnettle as you try picking it up!</span>")
+			to_chat(user, "<span class='userdanger'>You are stunned by the [src] as you try picking it up!</span>")
```

Comment:
I'm 99% sure this turns into 'the the deathnettle', since it's the entire object wich includes 'the'

Reply:
Good catch, fixed it **EVEN MORE**. I doubt this codebase can stand the sheer power of my fixing.